### PR TITLE
Removed circular dependancy in UMD definition.

### PR DESCRIPTION
**Pull request by ryantemple**
Based on notes at http://davidbcalhoun.com/2014/what-is-amd-commonjs-and-umd/ .  I believe this may solve the dependancy problem.

Works with the code given at https://github.com/jonrh/isjs-webpack-bugExample

### CODE REVIEW

File: is.js [hunk 1]
```diff
@@ -5,22 +5,22 @@
...
         // like Node.
-        module.exports = factory(require('is_js'));
+        module.exports = factory();
     } else {
         // Browser globals (root is window)
-        root.is = factory(root.is);
+        root.is = factory();
     }
-} (this, function(is) {
+} (this, function() {
```
Comment (sk-sparhandy) [inline is.js]: By deleting ```is``` from the parameter list of function ```is``` is not defined and needs to be declared by using a var-statement:

```
var is = {};
```

Comment (ryantemple) [inline is.js]: good catch - have updated this.

**Comment by rubenspgcavalcante** [comment]
:+1:

**Comment by arasatasaygin** [comment]
This was a buggy bug for a long time I think. Thanks for the help guys.

**Pull request merged**